roads.mss

```diff
@@ -2984,9 +2994,10 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       text-clip: false;
       text-placement: line;
       text-halo-radius: @standard-halo-radius;
-      [highway = 'living_street'] { text-halo-fill: @living-street-fill; }
+      [highway = 'living_street'] { text-halo-fill: @living-street-fill; text-repeat-distance: @major-highway-text-repeat-distance;}
```

Comment:
I think that text-repeat-distance for living street should be in separate line, so the brackets should create a small inner block.

Reply:
Could you fix this remaining issue? Than it would ready to merge.

Reply:
@kocio-pl : done.